posthog/session_recordings/queries/session_recording_list_from_filters.py

```diff
@@ -474,6 +484,9 @@ def _where_predicates(self) -> ast.Expr:
         if self.event_properties:
             exprs.append(property_to_expr(self.event_properties, team=self._team, scope="replay"))
 
+        if self._team.person_on_events_mode and self.person_properties:
```

Comment:
Can we reuse the same `poe_mode_active` as above?

Reply:
yeah, we totally could 🙈 

i'll follow up once i know if it works, i'm too excited :)